content/en/real_user_monitoring/guide/tracking-rum-usage-with-usage-attribution-tags.md

```diff
@@ -46,21 +54,51 @@ RumMonitor.shared().addAttribute(forKey: "department", value: "marketing")
 
 **Note**: A few tags are included by default (`service`, `env`, `version`, `application.id`, and `application.name`). For anything else, set the global context using the method above.
 
-Once you've deployed this step, new RUM sessions are tracked according to the tags you added.
+After you've deployed this step, new RUM sessions are tracked according to the tags you added.
 
 ## View RUM usage
-The newly tagged sessions are displayed on the [Usage Attribution][3] page. When you review the RUM with Session Replay Sessions and RUM Sessions columns, you can see the number of sessions by department.
+The newly tagged sessions are displayed on the [Usage Attribution][1] page. When you review the RUM with Session Replay Sessions and RUM Sessions columns, you can see the number of sessions by department.
 
 {{< img src="real_user_monitoring/guide/rum-usage-attribution-tags/rum-use-attribution-tags-3.png" alt="View RUM usage by department" style="width:100%;">}}
 
-Usage information is also available through the [`GetHourlyUsageAttribution`][4] endpoint.
+Usage information is also available through the [`GetHourlyUsageAttribution`][5] endpoint.
+
+## Enforce tags to be set at the org level
+
+{{< callout url="#" btn_hidden="true" header="false">}}
+  Enforced RUM Usage Attribution is in Preview. To request access, contact <a href="https://docs.datadoghq.com/help/"> Datadog Support</a>.
+{{< /callout >}}
+
+You can require that a usage attribution tag gets added to a RUM application so that you can understand how its usage contributes to your Datadog bill. When this setting is active, usage attribution tags are required to be set upon creating a new or updating an existing RUM application in Datadog per organization.
+
+If your Datadog environment has a parent and child organization and the setting is enforced in both organizations, you are required to set attribution tags for both. For example, if 3 attribution tags are required for the parent org and 2 attribution tags are required for the child org, you are prompted to set a total of 5 tags for each of your applications.
+
+1. Ensure you have the RUM Settings Write permission.
+2. Navigate to [**Digital Experience** > **Real User Monitoring** > **Manage Applications** > **Enforce Usage Attribution**][6].
```

Comment:
The link 404s.

Reply:
Ah right because it's behind a FF. Will remove link.